deploy/alerts/simple-comparison.yaml

```diff
@@ -0,0 +1,124 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: fixed-compares
```

Comment:
I think you'd set the same `prometheus: stf-default` as above.

Reply:
That doesn't matter.

Reply:
Again, consistency with how this would look when deploying via templates and potentially having multiple clouds, etc.